### PR TITLE
kola: add test for randomized disk UUID

### LINKED ISSUE #113: kola: add disk-uuid check
Need to make sure the disk uuid got randomized on first boot. The test in the AMI script was added here: https://github.com/coreos/scripts/commit/ae7e75954445f507dc8554e73d5ff20e6bb4294b

### PR DESCRIPTION
**Pull request by mischief**
fixes #113

### CODE REVIEW

**Comment by mischief** [comment]
@marineam PTAL

File: kola/tests/coretest/core.go [hunk 1]
```diff
@@ -214,6 +215,26 @@ func TestReadOnlyFs() error {
...
+	var out bytes.Buffer
+	c := exec.Command("sh", "-ec", "sudo blkid -o value -s PTUUID /dev/$(lsblk -no PKNAME $(findmnt -vno SOURCE /))")
+	c.Stdout = &out
+	err := c.Run()
+	if err != nil {
+		return fmt.Errorf("findmnt: %v", err)
+	}
+
+	defaultUUID := []byte("00000000-0000-0000-0000-000000000001")
+	got := bytes.TrimSpace(out.Bytes())
```
Comment (marineam) [inline kola/tests/coretest/core.go]: perhaps use the uuid library to actually parse the output so we know it dodn't fail in a weird way.

File: kola/tests/coretest/core.go [hunk 2]
```diff
@@ -214,6 +215,26 @@ func TestReadOnlyFs() error {
 	return fmt.Errorf("could not find /usr or / mount points.")
 }
 
+// Test that the root disk's UUID was set to a random one on first boot.
+func TestFsRandomUUID() error {
+	var out bytes.Buffer
+	c := exec.Command("sh", "-ec", "sudo blkid -o value -s PTUUID /dev/$(lsblk -no PKNAME $(findmnt -vno SOURCE /))")
+	c.Stdout = &out
+	err := c.Run()
```
Comment (marineam) [inline kola/tests/coretest/core.go]: could just use c.Output instead of bothering with a buffer yourself

**Comment by mischief** [comment]
@marineam ptal

**Comment by marineam** [comment]
LGTM

**Pull request merged**